### PR TITLE
[Automation] Making the AKS cluster name predictable

### PR DESCRIPTION
**Pull request by izaac**
Realized this should be predictable, I used the resource prefix from the HA automation

### CODE REVIEW

**Review by jiaqiluo — DISMISSED** [review]
lgtm

File: tests/validation/tests/v3_api/resource/terraform/aks/aks.tf [hunk 1]
```diff
@@ -11,7 +11,7 @@ resource "random_string" "random" {
 }
 
 resource "azurerm_resource_group" "default" {
-  name     = "testautoaks${random_string.random.result}"
+  name     = "testautoaks"
```
Comment (bmdepesa) [inline tests/validation/tests/v3_api/resource/terraform/aks/aks.tf:14]: Would there be a problem with multiple clusters in the same resource group?

Comment (izaac) [reply tests/validation/tests/v3_api/resource/terraform/aks/aks.tf:14]: I'll double check this and get back to you @bmdepesa

Comment (izaac) [reply tests/validation/tests/v3_api/resource/terraform/aks/aks.tf]: You're correct i reverted the resource name with a random suffix for now. Will figure out something for the delete ha for these two AKS/GKE cluster types later.

**Review by bmdepesa — APPROVED** [review]
LGTM

**Review by jiaqiluo — APPROVED** [review]
lgtm

**Review by sowmyav27 — APPROVED** [review]
LGTM

**Pull request merged**